comms/dht/src/storage/error.rs

```diff
@@ -30,6 +30,10 @@ pub enum StorageError {
     InvalidUnicodePath,
     #[error("ConnectionError: {0}")]
     ConnectionError(#[from] diesel::ConnectionError),
+    #[error("UniqueViolation")]
+    UniqueViolation(String),
+    #[error("ParseError")]
+    ParseError(String),
```

Comment:
Not needed (formatted_naive_date_time is not needed)

Reply:
Please see previous comment.

Reply:
Even if we keep the function, the error you return can never happen (it can never fail to parse because the input comes from a guaranteed well-formed string).

Reply:
Removed `ParseError(String)` along with `fn formatted_naive_date_time `